ext/puma_http11/extconf.rb

```diff
@@ -12,4 +12,10 @@
   end
 end
 
+BROKEN_VERSIONS = %w(2.2.7 2.3.4 2.4.1)
```

Comment:
Actually, because of the way this works, maybe we should only patch the latest teeny, because that's all we test on CI. This patch could potentially break on a teeny change (since we're so deep in the MRI internals here), so maybe targeting to *really specific* versions isn't a bad idea.